wire-ios-data-model/Tests/MLS/MLSClientManagerTests.swift

```diff
@@ -147,15 +164,23 @@ class MLSClientManagerTests: ZMBaseManagedObjectTest {
         // Given
         BackendInfo.isMLSEnabled = true
         let mlsFeature = mockFeatureRepository.fetchMLS()
-        await syncMOC.perform {
+        let domain = "example.domain.com"
+
+        let selfUser = await syncMOC.perform {
             let selfUser = ZMUser.selfUser(in: self.syncMOC)
-            selfUser.domain = "example.domain.com"
+            selfUser.domain = domain
+            return selfUser
         }
+
         let selfClient = await syncMOC.perform {
             self.createSelfClient(onMOC: self.syncMOC)
         }
+
         let qualifiedID = await syncMOC.perform {
-            selfClient.qualifiedClientID
+            QualifiedClientID(
+                userID: selfUser.remoteIdentifier,
+                domain: selfUser.domain ?? "",
```

Comment:
I would avoid falling back to empty strings unless it's an expected value

Reply:
fixed